openlibrary/templates/account/create.html

```diff
@@ -75,5 +78,31 @@ <h1>$_("Sign Up")</h1>
             <a href="javascript:history.go(-1);" class="smaller attn">$_("Cancel")</a>
         </div>
     </form>
-
-</div>
+    <style>
+        #password-visibility-toggle {
+            font-size: 14px;
+            margin-left: 5px;
+            cursor: pointer;
+        }
+
+    </style>
```

Comment:
```suggestion
```
We try to avoid adding inline styles whenever possible.  New rules for the password field should be added to https://github.com/internetarchive/openlibrary/blob/master/static/css/components/form.olform.less